manifests/python.yml

```diff
@@ -617,6 +617,12 @@ tests/:
         flask-poc: v2.11.0
         uds-flask: v2.11.0
         uwsgi-poc: v2.11.0
+    test_debugger_code_origins.py:
```

Comment:
@shurivich Should I be defining this in every manifest? In which cases is the `missing_feature` annotation sufficient versus defining the language/weblog as missing here?